libraries/AP_NavEKF3/AP_NavEKF3_Measurements.cpp

```diff
@@ -683,7 +683,7 @@ void NavEKF3_core::readAirSpdData()
 *              Range Beacon Measurements                *
 ********************************************************/
 
-// check for new airspeed data and update stored measurements if available
+// check for new range beacon data and update position if available
```

Comment:
Beacon data contains range and position information. Change to:

// check for new range beacon data push to data buffer if available